nhais-adaptor/adaptor/incoming/tests/test_incoming_adaptor.py

```diff
@@ -5,17 +5,16 @@
 from testfixtures import compare
 from utilities.file_utilities import FileUtilities
 
-from adaptor.incoming.operation_definition_adaptor import OperationDefinitionAdaptor
-import edifact.incoming.parser.deserialiser as deserialiser
 from adaptor.incoming.config import reference_dict
+from adaptor.incoming.incoming_adaptor import IncomingAdaptor
 
 
-class TestEdifactToFhirIntegration(unittest.TestCase):
+class TestIncomingAdaptor(unittest.TestCase):
     """
     Test that when fhir json payload is loaded the correct edifact message is created
     """
 
-    def test_patient_registration_birth_approval(self):
+    def test_covert_to_fhir(self):

```

Comment:
Small typo here - should be `convert`.

Reply:
Doh! when a typo is an actual word. Will update 👍